### PR TITLE
Fettec: Promote up one level in tree, shrink title image, fix two links

### PR DESCRIPTION
**Pull request by rmackay9**
This makes 3 small changes to the FETtecOne wiki page:

1. moves it up out of the "Serial ESC" page (which is then deleted).  I think we should avoid landing pages with just one item on them especially considering that the regular ESC-and-motors page still has lots of room on it.
2. fixes a couple of links on the page
3. removes white space from the title image

This has been tested on my local machine

### CODE REVIEW

**Comment by amilcarlucas** [comment]
I have very high hopes that there will be other serial based ESCs in the future, so I do not agree with the first change.
But the other two are good to go.

**Review by amilcarlucas — COMMENTED** [review]
I have very high hopes that there will be other serial based ESCs in the future, so I do not agree with the first change.
But the other two are good to go.

Thanks Randy

**Comment by rmackay9** [comment]
@amilcarlucas,

We can always add the layer back if required in the future.  .. none of this is written in stone.. and even if we did have more serial ESCs why would we want to hide them one more layer down?

File: common/source/docs/common-fettec-onewire.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ FETtec OneWire ESCs
 
 .. image:: ../../../images/FETtec_4in1_ESC.jpg
```
Comment (Hwurzburg) [inline common/source/docs/common-fettec-onewire.rst:9]:
```suggestion
.. image:: ../../../images/FETtec_4in1_ESC.jpg
    :target: ../../_images/FETtec_4in1_ESC.jpg
    :width: 450px
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
dont delete image and replace with lower resolution

**Comment by rmackay9** [comment]
@Hwurzburg,

I don't think I made it lower resolution.. I just chopped off the white space around the important part of the image.

File: common/source/docs/common-fettec-onewire.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-fettec-onewire.rst:9]: you're right...sorry

**Pull request merged**